contrib/completion/bash/docker

```diff
@@ -4391,12 +4437,12 @@ _docker_stack() {
 }
 
 _docker_stack_deploy() {
+	__docker_complete_stack_orchestrator_options && return
+
 	case "$prev" in
 		--bundle-file)
```

Comment:
I removed the selective completion here although it is correct behavior.

The important part is that `--bundle-file` is not completed if it is not appropriate, which is already taken care for.
If someone insists on manually typing this option, he will now get it completed although it does not exist in this context.
This is an error, but given the growing complexity of hiding options based on operating system, client/server experimental state and orchestrator replicating the hiding logic in the individual option's completions will bloat the code without much value.